src/Pages/SignIn.tsx

```diff
@@ -1,11 +1,95 @@
-import React from 'react';
+import { FormEvent, useRef, useState } from 'react';
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import { Typography, Button, Alert } from '@mui/material';
+import { Facebook } from '@mui/icons-material';
+import '../Assets/FormStyle.css'
+import { useNavigate } from 'react-router-dom';
+import { useAuth } from "../authContext";
+import GoogleButton from 'react-google-button';
 
 
 function SignInPage() {
-  return (
-    <div>
-    </div>
-  );
+	const { login, loginEmail, setLoginEmail, loginPassword, setLoginPassword, googleSignIn} = useAuth();
+	const [error, setError] = useState("");
+	const [loading, setLoading] = useState(false);
+	const navigate = useNavigate();
+
+	const handleSubmit = async (e: FormEvent) => {
+		e.preventDefault();
+
+		try {
+		await login(loginEmail, loginPassword);
+		console.log("login successful");
```

Comment:
Should we have this console log? Maybe we can keep it for now but remove it once we have a sign in succeeded confirmation

Reply:
I will edit and add the password confirmation as well before we merge 🦘

Reply:
password confirmation is already added! :)

Reply:
To firebase? I added it just now